contracts/kleros/KlerosGovernor.sol

```diff
@@ -171,7 +171,7 @@ contract KlerosGovernor is Arbitrable {
         executionTimeout = _executionTimeout;
     }
 
-    /** @dev Changes the time allowed for list withdrawal.
+    /** @dev Changes list withdrawal timeout. Note that the withdrawal period is the intersecion between the withdraw timeout and the first half of the submission period.
```

Comment:
This makes it sound more complicated than it actually is. I'd rather not add anything or at least rephrase it to "Note that withdrawals are only possible in the first half of the submission period"

Also, "intersecion" -> "intersection"